.github/.gitignore

```diff
@@ -0,0 +1 @@
+*.yml
```

Comment:
General comment in a changed line.

Reply:
Starting a review in this specific line (I do not know what it means exactly).

Reply:
bla